src/index.js

```diff
@@ -1,6 +1,16 @@
 const fibonacci = (n) => {
-
+  let answer=[n-1]
+  let e=1, r=0,i,c=0
+  while(n>0){
```

Comment:
En este caso tienes un número conocido de veces que harás el ciclo, por lo que es mejor usar for.